src/Ombi.Settings/Settings/Models/LdapSettings.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Ombi.Settings.Settings.Models
+{
+    public class LdapSettings : Settings
+    {
+        public LdapSettings()
+        {
+            IsEnabled = false;
```

Comment:
Can we please remove this constructor, if we want to assist the user and pre-populate the settings then we should do that in the UI